geode-wan/src/main/java/org/apache/geode/cache/client/internal/locator/wan/LocatorHelper.java

```diff
@@ -100,6 +129,14 @@ public static boolean addExchangedLocators(Map<Integer, Set<DistributionLocatorI
           if (!localLocators.equals(entry.getValue())) {
             entry.getValue().removeAll(localLocators);
             for (DistributionLocatorId locator : entry.getValue()) {
+              if (!locator.getMemberName().equals(DistributionConfig.DEFAULT_NAME)
```

Comment:
would you please add a comment here as to why it's okay to ignore the other map's locator ID when the membername matches one that's already in the local map?